selfdrive/car/toyota/carcontroller.py

```diff
@@ -99,19 +99,21 @@ def update(self, CC, CS, now_nanos):
                                                           lta_active, self.frame // 2, torque_wind_down))
 
     # *** gas and brake ***
-    if self.CP.enableGasInterceptor and CC.longActive:
+    if self.CP.enableGasInterceptor and CC.longActive and self.CP.carFingerprint not in STOP_AND_GO_CAR:
       MAX_INTERCEPTOR_GAS = 0.5
       # RAV4 has very sensitive gas pedal
-      if self.CP.carFingerprint in (CAR.RAV4, CAR.RAV4H, CAR.HIGHLANDER):
+      if self.CP.carFingerprint == CAR.RAV4:
```

Comment:
This was modified because `CAR.RAV4H` and `CAR.HIGHLANDER` are both marked as `stop_and_go = True` in interface.py